bootstrap.lisp

```diff
@@ -0,0 +1,25 @@
+;;;; Creates executable for linux and windows.
+;;; usage: sbcl --load bootstrap.lisp
+;;; emacs: M-! sbcl --load bootstrap.lisp
+
```

Comment:
I think you should add `(in-package :cl-user)` to the top of the file.
To be sure that all functions and variables are taken from proper package.